analytics_data_api/v0/views/learners.py

```diff
@@ -60,6 +60,7 @@ class LearnerView(LastUpdateMixin, CourseViewMixin, generics.RetrieveAPIView):
         Returns metadata and engagement data for the learner in JSON format.
 
             * username: The username of the enrolled learner.
+            * account_url:  URL to learner's account page.
```

Comment:
This is actually the URL to the learner's endpoint on the account API. There's probably a better way to word it than I just did, but I think it's important to distinguish between a page (e.g. HTML document) and API (e.g. JSON string).